services/ingestion/engine.go

```diff
@@ -181,18 +181,11 @@ func (e *Engine) processBlockEvent(cadenceHeight uint64, event cadence.Event) er
 }
 
 func (e *Engine) processTransactionEvent(event cadence.Event) error {
-	tx, err := models.DecodeTransaction(event)
+	evmTxData, err := models.DecodeTransaction(event)
```

Comment:
no need to change this, they are still transactions. I would avoid changing all the transactions to tx data. I think direct call is just one sub-type of a transaction.